README.md

```diff
@@ -13,8 +19,19 @@ Auditing Reporter for Greener Ops Strategies
 
 ## Usage
 
-`NAME="[name_of_the_run]" docker-compose up`
+### start the app
+
+`make start` start the server and the mongo database.
+
+Go to localhost:3003 to see the report.
+
+### Run the cli
+
+The cli allows to realize measure. It will execute the given command x times, and then compute the average, min and max of the realized measures.
```

Comment:
```suggestion
The CLI allows to realize measures. It will execute the given command x times, and then compute the average, min and max of the realized measures.
```